app/views/list/_request_row_template.slim

```diff
@@ -0,0 +1,95 @@
+tr.request
+  td.request-checkbox
+    input[type="checkbox"]
+  td.request-edit
+    a[href="#"]
+      | Edit
+  td.request-name
+    a[href="#"]
+      | Rubik's Cube Game
+  td.request-recipient
+    | Mark
+  td.request-status
+    | Posted
+  td.request-age
+    | 9
+  td.request-gender
+    | M
+  td.request-description
+    | Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed ... 
+tr.request
```

Comment:
The partial for a single row should only have one `tr` in it - this will be easy to resolve by making the partial dynamic.